scripts/linux/script.sh

```diff
@@ -140,6 +137,9 @@ rsync -a --exclude='.*' ${RSYNC_EXCLUDE_DIRS} . .tmp/${WORKDIR} || die "Failed"
 print G "done."
 cd .tmp
 
+print Y "Importing translation files..."
```

Comment:
IMO, we shouldn't be calling this script directly anymore, it will dirty the source directory with the translation build artifacts and everything (except the submodule updates) are now handled by CMake at build time.

However, the script still needs to exist for iOS because it's still stuck on qmake.